Shared.Tests/CloudinaryTest.cs

```diff
@@ -2220,6 +2220,68 @@ public void TestUpdateTransformUnsafe()
             Assert.AreEqual(updateParams.UnsafeTransform.Generate(), new Transformation(getResult.Info).Generate());
         }
 
+        [Test]
+        public void TestUpdateTransformWithExtension()
+        {
+            // should allow updating a transformation string contains a format extension
+
+            string transformationName = m_updateTransformationWithExtAsString;
+
+            m_cloudinary.CreateTransform(
+                new CreateTransformParams()
+                {
+                    Name = transformationName,
+                    Transform = m_simpleTransformation,
+                    Format = m_extension
+                });
+
+            var updateParams = new UpdateTransformParams()
+            {
+                Transformation = transformationName,
+                UnsafeTransform = m_updateTransformation,
+                Strict = true,
+                Format = m_extension
+            };
+
+            m_cloudinary.UpdateTransform(updateParams);
+
+            var getResult = m_cloudinary.GetTransform(transformationName);
+
+            Assert.IsNotNull(getResult.Info);
+            Assert.AreEqual(updateParams.UnsafeTransform.Generate(), new Transformation(getResult.Info).Generate());
+        }
+
+        [Test]
+        public void TestUpdateTransformWithEmptyExtension()
```

Comment:
Maybe try to unify 3 versions of the same test, you have 80% of common code here

Reply:
I've merged the two I've created to 'TestUpdateTransformWithImplicitExtension'.